apps/somnia/src/commands/archive/weapon.ts

```diff
@@ -61,13 +90,20 @@ export function weaponArchive(
     //name and passive
     name += ` (R${refinedisplay[refine]})`
     text +=
-      `\n\n**${data.passiveName}:** ` +
-      Object.values(data.passiveDescription[refine]).join('\n')
+      '\n\n**' +
```

Comment:
prob should refactor this and the other text+= into a readable function
`text += passiveNameAndDesc()` for example